modules/base/manifests/ntp.pp

```diff
@@ -0,0 +1,20 @@
+class base::ntp (
```

Comment:
If you want to manage NTP so do it in separate class in separate module, otherwise just install ntpd and ensure that service is running. Also as I've mentioned before I might be a good idea to get rid of BASE class at all.